src/status_im2/contexts/chat/messages/content/album/view.cljs

```diff
@@ -0,0 +1,28 @@
+(ns status-im2.contexts.chat.messages.content.album.view
+  (:require [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]
+            [status-im2.contexts.chat.messages.content.album.style :as style]
+            [status-im2.common.constants :as constants]))
+
+
+(defn album-message
+  [message]
+  [rn/view
+   {:style style/album-container}
+   (map-indexed (fn [index item]
+                  (let [images-count    (count (:album message))
+                        images-size-key (if (< images-count 6) images-count :default)
+                        size            (get-in constants/album-image-sizes [images-size-key index])]
+                    [rn/view {:key (:message-id item)}
+                     [fast-image/fast-image
+                      {:style  (style/image size index)
+                       :source {:uri (:image (:content item))}}]
+                     (when (and (> images-count 6) (= index 5))
```

Comment:
Would be better if we parameterized the magic number `6`. If, in the future, this number changes from 6 to 7 for instance, we should just have to change the constant value.

Reply:
Parameterized the magic number 😄